libexec/bats-core/bats-exec-suite

```diff
@@ -120,6 +120,12 @@ if [[ "${test_count}" == 0 ]]; then
   exit
 fi
 
+if ! BATS_SUITE_TMPDIR=$(mktemp -d "${BATS_RUN_TMPDIR}/bats-suite-tmpdir-XXXXXX");then
```

Comment:
I think the `$BATS_RUN_TMPDIR` prefix gives us enough scope to omit the bats- prefix on lower levels, so `${BATS_RUN_TMPDIR}/suite-tmpdir-XXXXX` should be enough.